packages/app/src/app/overmind/factories.ts

```diff
@@ -80,3 +80,78 @@ export const withOwnedSandbox = <T>(
 
   return continueAction(context, payload);
 };
+
+export const createModals = <
+  T extends {
+    [name: string]: {
+      state?: IState;
+      result?: any;
```

Comment:
Maybe use `unknown` type instead too add? 🤔

Reply:
Yeah, agree on unknown here is it affects the DX :)